### PR TITLE
Add another deadline item

### PR DESCRIPTION
**Pull request by jonathangoulding**

### CODE REVIEW

File: packages/forms-web-app/src/config.js [hunk 1]
```diff
@@ -66,8 +66,10 @@ module.exports = {
 				description: 'description',
 				email: 'email',
 				hasInterestedPartyNo: 'hasInterestedPartyNo',
+				comment: 'comment',
 				id: 'id',
```
Comment (christopher-semmens) [inline packages/forms-web-app/src/config.js:70]: Are these values being used. Are we not moving away from this config?

**Review by christopher-semmens — APPROVED** [review]

File: packages/forms-web-app/src/config.js (hunk 1 above)
Comment (jonathangoulding) [reply packages/forms-web-app/src/config.js:70]: addressed in next PR for removal

**Pull request merged**